src/makeRobot.js

```diff
@@ -39,6 +39,60 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goForward(step = 1) {
+      if (step < 0) {
+        return this;
+      }
+      this.coords.y += step;
```

Comment:
```suggestion
      if (step > 0) {
        this.coords.y += step;
      }
```
і не треба тоді 2 return